integrations/docker/images/stage-2/chip-build-nuttx/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG VERSION=1
+FROM ghcr.io/apache/nuttx/apache-nuttx-ci-linux:${VERSION}
```

Comment:
I am a bit weary of this because:

- it will not copy over in vscode so other developers will have a hard time testing nuttx
- chip-build seems to install other bits that may be needed (sets python3.9, installs various things into it). It is not minimal however it is somewhat nice that we know what our activate/python scripts can depend on.

the {VERSION} here is the chip-build version and does not seem related to apache-nuttx-ci-linux version.

Reply:
@andy31415 Ok, I will try to create a docker image that can successfully compile nuttx binary based on base chip-build. Thanks.